### DISCUSSION TITLE
How to Add support for older codecs within the app ?

### DISCUSSION THREAD

**Discussion by KAPILBADWAL** [jhomlala/betterplayer#697] [Q&A] [answered]
Hi,  having a great experience using better player. But I am having this issue where a certain video is able to play on one device and not on another. I concluded it to missing codecs in other devices. My video uses Mpeg 1/2 Video and ADTS audio.
Is there any way I can add the codec support in the player itself so that to make it device independent ?

**Comment by jhomlala** [comment, marked as answer]
It's not possible right now. If you want to support it only on Android, then you can try to fork BP and add ExoPlayer ffmpeg extension.

**Reply by KAPILBADWAL** [reply]
Ok. Thanks
